### ISSUE TITLE
UTF-8 Encoding (Umlauts)

### ISSUE THREAD

**Issue by benedikt-bartscher** [rhasspy/rhasspy#111]
Some german words have Umlauts (äöü) and other UTF-8 Characters (ß for example). Rhasspy successfully recognizes them and sends them via MQTT as such:
```
mosquitto_sub -h mqtt-snips -t 'hermes/asr/#' -v
hermes/asr/textCaptured {"text": "k\u00fcche einschalten", "likelihood": 1, "seconds": 2.1554993869999635, "siteId": "default", "sessionId": "default-jarvis-032c9803-e4e4-4c25-a8f8-751dc02c770c", "wakewordId": null, "asrTokens": null, "lang": null}
```

I expect the output to be like this:
```
mosquitto_sub -h mqtt-snips -t 'hermes/asr/#' -v
hermes/asr/textCaptured {"text": "küche einschalten", "likelihood": 1, "seconds": 2.1554993869999635, "siteId": "default", "sessionId": "default-jarvis-032c9803-e4e4-4c25-a8f8-751dc02c770c", "wakewordId": null, "asrTokens": null, "lang": null}
```

I used Snips before, with Snips the Umlauts got correctly encoded and published via MQTT.

**Comment by synesthesiam** [comment]
OK, I see where the problem is. Python's [json.dump](https://docs.python.org/3/library/json.html) has a flag `ensure_ascii` set to `True` by default, which escapes everything non-ASCII. I'll need to patch all the places that call `json.dump`.

In the mean time, I've found that just passing JSON through [jq](https://stedolan.github.io/jq/) will fix it:

```sh
$ echo '{"text": "k\u00fcche einschalten"}' | jq .
{
  "text": "küche einschalten"
} 
```

**Comment by benedikt-bartscher** [comment]
Version 2.5.6 Changelog mentions this problem, but for me it's still present.

**Comment by synesthesiam** [comment]
I've tracked down this problem to the JSON serialization library we use for the Hermes messages. I'll bundle this and some other bug fixes into a update to 2.5.6 this week.

**Comment by benedikt-bartscher** [comment]
Should this be fixed by Version 2.5.7?

**Comment by synesthesiam** [comment]
That's the plan now. The bug fixes for 2.5.6 are more substantial, so I'm just going to roll them into 2.5.7.

**Comment by benedikt-bartscher** [comment]
No problems since 2.5.7. Thank you :+1:
